### PR TITLE
Pass ampcontext file path down on iframe name with other attributes.

### PR DESCRIPTION
**Pull request by bradfrizzell**
As requested in https://github.com/ampproject/amphtml/issues/6829#issuecomment-269538913

### CODE REVIEW

File: src/iframe-attributes.js [hunk 1]
```diff
@@ -51,6 +51,9 @@ export function getContextMetadata(
       parentWindow, 'sentinel-name-change');
   attributes._context = {
     ampcontextVersion: '$internalRuntimeVersion$',
+    ampcontextFilepath: 'https://cdn.ampproject.org/' +
```
Comment (jridgewell) [inline src/iframe-attributes.js]: This needs to use [`urls.cdn`](https://github.com/ampproject/amphtml/blob/master/src/config.js#L34).

Comment (bradfrizzell) [inline src/iframe-attributes.js]: done

File: test/functional/test-3p-frame.js [hunk 2]
```diff
@@ -174,6 +174,9 @@ describe('3p-frame', () => {
           '"type":"_ping_",' +
           '"_context":{"referrer":"http://acme.org/",' +
           '"ampcontextVersion": "$internalRuntimeVersion$",' +
+          '"ampcontextFilepath": "https://cdn.ampproject.org/' +
+          '$internalRuntimeVersion$' +
+          '/ampcontext-v0.js",' +
```
Comment (lannka) [inline test/functional/test-3p-frame.js]: move it to the previous line?

**Comment by lannka** [comment]
@bradfrizzell any doc update needed? Or it has to wait until we fully migrated to the new context API?

**Comment by bradfrizzell** [comment]
The intent to implement thread already reflects that this change is being made. These is no consolidated documentation anywhere else that needs a change.

**Comment by lannka** [comment]
oh right, user doesn't need to know about the resource path.

**Pull request merged**